interpreter/cling/lib/Interpreter/BackendPasses.cpp

```diff
@@ -344,36 +339,12 @@ BackendPasses::~BackendPasses() {
   //delete m_PMBuilder->Inliner;
 }
 
-void BackendPasses::CreatePasses(llvm::Module& M, int OptLevel)
-{
-  // From BackEndUtil's clang::EmitAssemblyHelper::CreatePasses().
-
-#if 0
-  CodeGenOptions::InliningMethod Inlining = m_CGOpts.getInlining();
-  CodeGenOptions& CGOpts_ = const_cast<CodeGenOptions&>(m_CGOpts);
-  // DON'T: we will not find our symbols...
-  //CGOpts_.CXXCtorDtorAliases = 1;
-
-  // Default clang -O2 on Linux 64bit also has the following, but see
-  // CIFactory.cpp.
-  CGOpts_.DisableFPElim = 0;
-  CGOpts_.DiscardValueNames = 1;
-  CGOpts_.OmitLeafFramePointer = 1;
-  CGOpts_.OptimizationLevel = 2;
-  CGOpts_.RelaxAll = 0;
-  CGOpts_.UnrollLoops = 1;
-  CGOpts_.VectorizeLoop = 1;
-  CGOpts_.VectorizeSLP = 1;
-#endif
-
-#if 0 // def __GNUC__
-  // Better inlining is pending https://bugs.llvm.org//show_bug.cgi?id=19668
-  // and its consequence https://sft.its.cern.ch/jira/browse/ROOT-7111
-  // shown e.g. by roottest/cling/stl/map/badstringMap
-  if (Inlining > CodeGenOptions::NormalInlining)
-    Inlining = CodeGenOptions::NormalInlining;
-#endif
-
+void BackendPasses::CreatePasses(llvm::Module& M, int OptLevel,
+                                 llvm::ModulePassManager& m_MPM,
```

Comment:
please rename to
```suggestion
                                 llvm::ModulePassManager& MPM,
```
the `m_` prefix is for member variables.